src/components/Dropdown/LanguageDropdown/LanguageDropdown.tsx

```diff
@@ -4,14 +4,21 @@ import Globe from '../../../../public/icons/globe.svg'
 import { LocaleContext } from '../../../contexts/LocaleContext'
 import { Language, locales } from '../../../types/index'
 import { localeToLanguage, languageToLocale } from '../../../utils/utils'
+import Cookies from 'js-cookie'
 
 export default function LanguageDropdown() {
   const { setLocale, locale } = useContext(LocaleContext)
 
   const handleOnClick = (e: React.ChangeEvent<HTMLSelectElement>) => {
-    return setLocale
-      ? setLocale(languageToLocale(e.currentTarget.value as Language))
-      : null
+    return (
+      setLocale
+        ? setLocale(languageToLocale(e.currentTarget.value as Language))
+        : null,
+      Cookies.set(
+        'Langugae',
```

Comment:
typo?

Reply:
Yes, it is. God catch thank you.
That code is actually not necessary anymore so I will remove it.